python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -33,6 +33,11 @@ class SnowflakeConnection:
     """
 
     def __init__(self, config: Mapping[str, str], log):
+        config = cast(Dict[str, str], config)
+        # with pydantic resources the schema config field will be named schema_
+        if config.get("schema_") is not None:
+            config["schema"] = config["schema_"]
+
```

Comment:
I don't think this should be necessary.

```
from dagster import Config
from pydantic import Field


class ConfigWithSchema(Config):
    schema_: str = Field(alias="schema")


assert ConfigWithSchema(schema="foo")._as_config_dict() == {"schema": "foo"}
```

Reply:
it was failing before, but that was before ben got the alias piped all the way through, so it's probably good now. thanks for the catch!